src/bindings/capi/thorvg_capi.h

```diff
@@ -1339,6 +1339,24 @@ TVG_API Tvg_Result tvg_shape_get_stroke_gradient(const Tvg_Paint* paint, Tvg_Gra
 TVG_API Tvg_Result tvg_shape_set_stroke_dash(Tvg_Paint* paint, const float* dashPattern, uint32_t cnt);
```

Comment:
Please kill the old version of the api.  I think we don't need to maintain both apis.